src/vrv.cpp

```diff
@@ -84,37 +84,45 @@ bool Resources::InitFonts()
         return false;
     }
 
+    if (!SetTextFont("VerovioText-1.0", false)) {
+        LogError("Text font could not be initialized.");
+        return false;
+    }
+
+    s_currentStyle = k_defaultStyle;
+
+    return true;
+}
+
+bool Resources::SetMusicFont(const std::string &fontName)
+{
+    return LoadFont(fontName);
+}
+
+bool Resources::SetTextFont(const std::string &fontName, const bool usePostfixes)
+{
     struct TextFontInfo_type {
         const StyleAttributes m_style;
-        const std::string m_fileName;
+        const std::string m_postfix;
```

Comment:
What does `postfix` mean? Looks like a weird name change.

Reply:
Previous since it was hardcoded it was referring to the filename (e.g. Times-italic). Now that we can change text font by option, this refers to the postfix for the weight/style combination that should be added to the text font name when doing InitTextFont(). We have postfixes like -bold, -italic, -bold-italic and when init font (for instance, Arial) we get file following file names: Arial-bold, Arial-italic, etc.